integration/README.md

```diff
@@ -55,7 +55,7 @@ The client behavior will be:
 
 > TODO: For Go we will use dep
 
-Java will pin to the latest published version at the time and use Gradle for building
+Java will pin to the latest published version at the time and use Gradle for building. Note that the Java interop test client/server is only compatible up to JDK 8 (not compatible with JDK 9).
```

Comment:
Why is the interop test incompatible with Java 9?

Reply:
Apparently the gradlew script cannot "determine the version" 
```shell
* What went wrong:
Could not determine java version from '9.0.4'.
```
who knows what else lurks? :)

Reply:
sourceCompatibility and targetCompatibility are set to 1.8:
https://github.com/census-ecosystem/opencensus-experiments/blob/master/integration/java/build.gradle#L53

Which is the same as the [Java example build](https://github.com/census-instrumentation/opencensus-java/blob/master/examples/build.gradle#L32).

Reply:
I think that Java 9 should still support source and target compatibility with Java 8.  I would expect a different warning, like this warning from opencensus-java: https://travis-ci.org/census-instrumentation/opencensus-java/jobs/382426740#L1617-L1618

Do the Gradle scripts need to be upgraded?

Reply:
The script that caused this error: 
```shell
cd java && ./gradlew installDist && cd ..
```
To me it looks like it's correct (since we need to use the opencensus version from maven local). Do you have any idea how this could go wrong?

Reply:
It looks like Gradle couldn't parse the Java 9 version format until version 4.2.1: https://github.com/gradle/gradle/issues/2992#issuecomment-332869508.  This repository is only using Gradle 3.2.1, so I think we just need to upgrade it.

Reply:
Exactly, I tried upgrading to the latest gradle version and it worked fine with JDK 9. Thanks for looking into this!

Closing this PR, post https://github.com/census-ecosystem/opencensus-experiments/pull/37 instead.